bin/handler-slack.rb

```diff
@@ -189,6 +189,7 @@ def payload(notice)
 
     {
       icon_url: slack_icon_url ? slack_icon_url : 'https://raw.githubusercontent.com/sensu/sensu-logo/master/sensu1_flat%20white%20bg_png.png',
+      text: [slack_message_prefix, notice].compact.join(' '),
```

Comment:
won't this result in duplication of the same text?

Reply:
After checking again with both slack and mattermost: yes, you are right, it will be duplicated. It does solve the problem in mattermost, as it only processes mention notifications from the plain text version. This is a problem with mattermost, so i'll close this PR, and i'll use [my fork](https://github.com/linkorb/sensu-plugins-slack) with duplicate content until mattermost processes mentions from the html body.

Reply:
How about we have a `--matermost` option and then we can use this to hide any mattermost specific logic behind the flag.

Reply:
it'd still duplicate the text, that's not specific to mattermost. so a better flag-name would be `--duplicate-attachment-text-as-plain-text`. i think this is mostly a problem with mattermost (it should notify on mentions in formatted text, like slack does) that doesn't need to be solved by this plugin.